ibm/provider.go

```diff
@@ -517,6 +517,16 @@ func Provider() *schema.Provider {
 			"ibm_en_topics":        dataSourceIBMEnTopics(),
 			"ibm_en_subscription":  dataSourceIBMEnSubscription(),
 			"ibm_en_subscriptions": dataSourceIBMEnSubscriptions(),
+
+			// Compliance Posture Management v2
```

Comment:
```
ibm_scc_posture_scopes
ibm_scc_posture_latest_scans
ibm_scc_posture_profiles
ibm_scc_posture_scan_summary
ibm_scc_posture_scan_summaries
```
			
We already support datasources for above, just migrate existing datasources functions from v1 to v2
below datasources can be removed

```
ibm_scc_posture_v2_list_scopes
ibm_scc_posture_v2_list_profiles
ibm_scc_posture_v2_list_latest_scans
ibm_scc_posture_v2_scans_summary
ibm_scc_posture_v2_scan_summaries
```


Please remove `_v2` from  resources and datasources just to be inline with other sccc resources and datasources..
Please rename all file names and make respective changes in docs accordingly

```
`ibm_scc_posture_v2_profileDetails` ==>`ibm_scc_posture_profile`
`ibm_scc_posture_v2_group_profile_details` ==>`ibm_scc_posture_group_profile`
`ibm_scc_posture_v2_scope_correlation` ==> `ibm_scc_posture_scope_correlation`
```
please add near [already existing scc datasources](https://github.com/IBM-Cloud/terraform-provider-ibm/blob/a0d7985697370c22c1991ae74d4ad8e9c66517fa/ibm/provider.go#L507) so that, all scc related datasources are grouped at one place